src/gui/editors/PianoRoll.cpp

```diff
@@ -741,10 +741,10 @@ void PianoRoll::fitNoteLengths(bool fill)
 {
 	if (!hasValidMidiClip()) { return; }
 	m_midiClip->addJournalCheckPoint();
+	m_midiClip->rearrangeAllNotes();
```

Comment:
Any reason for this addition (I'm assuming you are calling ``m_midiClip->rearrangeAllNotes`` in place of the ``std::sort``?)

Reply:
I imagine it's because `refNotes` is now a const reference, so can't be sorted.